src/wagtail_ai/migrations/0002_populate_default_prompts.py

```diff
@@ -0,0 +1,36 @@
+# Created using Django 4.2.7 on 2023-12-08 09:39
+
+from django.db import migrations
+
+from wagtail_ai.prompts import DEFAULT_PROMPTS
+
+
+def set_default_ai_prompts(apps, schema_editor):
+    """
+    Loop through each default prompt and
+    populate the database.
+    """
+    Prompt = apps.get_model("wagtail_ai", "Prompt")
+    for default_prompt in DEFAULT_PROMPTS:
+        Prompt.objects.update_or_create(
+            default_prompt_id=default_prompt["default_prompt_id"],
```

Comment:
**Issue:** `prompt=None` is probably part of the lookup key too as we don't want to update ones that have a custom prompt, right?